### ISSUE TITLE
Cannot Select DATA Disk or DNS during setup

### ISSUE THREAD

**Issue by mezimmermann** [nextcloud/vm#968]
## Steps to reproduce
1. Setup Ubuntu latest LTS version with static IP in corporate environment
2. Execute nextcloud_install_production.sh
3. Create new user ncadmin, try get to the step to choose DATA path or DNS

## Expected behaviour
I can choose, where DATA should get stored (The test VM has a 10 GB Disk for system and a 50 GB DATA drive)
I also should be able to choose DNS provider.

## Actual behaviour
Empty list in disk selection 
Empty list in DNS selection 
Script continues when selecting ok but stopps at "specify a better mirror y/n" ---> input not possible.

## Server configuration

**Nextcloud server version:** (see your admin page)
Latest I guess, didn't managed to get this far.
**Server OS** (Ubuntu server is default)
Ubuntu Server 18.04.3 LTS (latest)
**How did you install the VM?** (Scripted install from master **OR** Released version)
basically according to the installation instructions 
https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-server#0
after installing pristine ubuntu, I patched the server and that's really it.
## Network
**Do you use DHCP?**
No, its a static configuration in a corporate environment.
**Is port 80 and/or 443 open?**
Yes. I'm able to update/pull from archives etc.

## Logs / Screenshots
didn't managed to get this far.

#### Log file (/mnt/ncdata/nextcloud.log)
didn't managed to get this far.
Insert your nextcloud.log file here
didn't managed to get this far.

#### Screenshoots
https://imgur.com/a/cFhJcGU

**Comment by mezimmermann** [comment]
Nevermind, works when pulling directly instead of creating script on my own
sudo bash -c "$(wget -q -O - https://raw.githubusercontent.com/nextcloud/vm/master/nextcloud_install_production.sh)"
